addons/website/models/theme_models.py

```diff
@@ -11,6 +11,13 @@
 
 _logger = logging.getLogger(__name__)
 
+class ThemeDesign(models.Model):
+    _inherit = 'website.design'
+    _name = 'theme.website.design'
+    _description = 'Website Theme Design'
+
+    theme_id = fields.Many2one('theme.utils', required=True)
```

Comment:
- `theme.utils` is an abstract model, it has no record. Making a many2one using that model does nothing
- `theme.utils` is a model that will be gone after your PR anyway
- The other `theme.XXX` do not have a `theme_id` field -> you are doing something wrong
- I don't think caring about themes should be your first step. I may be wrong but in my mind this can really be the very **last** step to implement